### PR TITLE
[search] Added FeatureOffsetsTable generation for test mwms.

### PR DESCRIPTION
**Pull request by ygorshenin**
https://trello.com/c/5JGlNyib/1919-search-integration-tests-master

### CODE REVIEW

**Comment by mapsmetest** [comment]
Jenkins asks: can anyone have a look at this request?
If you want Jenkins to compile and test this, write "test this please" or JT_TP (without the underscore in the middle).

File: indexer/features_offsets_table.hpp [hunk 1]
```diff
@@ -112,6 +112,6 @@ namespace feature
     detail::MappedFile::Handle m_handle;
   };
 
+  // Builds feature offsets table in an mwm.
```
Comment (mpimenov) [inline indexer/features_offsets_table.hpp]: or rebuilds an existing one?

Comment (mpimenov) [inline indexer/features_offsets_table.hpp]: Или этот метод ничего не делает, если таблица уже есть?

**Comment by ygorshenin** [comment]
PTAL

**Comment by mpimenov** [comment]
LGTM

**Pull request merged**